### DISCUSSION TITLE
Proposed changes to the functioning of the AC

### DISCUSSION THREAD

**Discussion by rkoshak** [openhab/openhab-distro#1499] [Architecture Council Discussions]
There is a thread that I think I'm the only AC member participating with a discussion on what the AC is doing (or not doing), what the policy says it should be doing and other AC related things. A lot of good points are raised including the fat that the verbiage in the policy that describes what the AC does and the description to the left on this page do not match and the policy makes it sound like the AC is much more involved in defining the architecture and future direction of OH.

https://community.openhab.org/t/increase-openness-and-competition-on-components-developed-outside-the-official-source-code-repositories/135183/9

I think it will be important that the AC weighs in on this topic to clarify, reassure, and address some of the concerns raised. Our silence I think will speak volumes and it will not be a good look. 

There is also a discussion on making it easier to support developer's ability to build add-ons without so much extra stuff which is the real focus of that thread. So maybe we should move the AC discussion to another thread (assuming you all agree that we should engage).

**Comment by kaikreuzer** [comment]
Imho this thread pretty much went off-topic and I find it hard to see what further comments could help there. I love your calm way to give precise and detailed answers. The OT has been solved meanwhile by moving it to an Github issue, so if anyone wants to follow-up on the AC discussion, I guess that topic should be split.
